common/brave_switches.cc

```diff
@@ -55,4 +55,7 @@ const char kFastWidevineBundleUpdate[] = "fast-widevine-bundle-update";
 
 const char kUseGoUpdateDev[] = "use-dev-goupdater-url";
 
+// Use custom update interval in sec
+const char kComponentUpdateInterval[] = "component-update-interval";
```

Comment:
If you add "seconds" to the variable name and the switch, users would immediately understand which units to use :)